### PR TITLE
feat: adds more informative package README

### PR DESCRIPTION
**Pull request by ixahmedxi**

### CODE REVIEW

File: packages/client/README.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+## Installation
+
+```bash
+# NPM
+npm install @trpc/client
+
+# Yarn
+yarn add @trpc/client
+
+# Pnpm
```
Comment (sachinraja) [inline packages/client/README.md:31]:
```suggestion
# pnpm
```

File: packages/client/README.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+> The `@trpc/client` package is responsible for communicating with a tRPC server on the client side.
+
+## Documentation
+
+Full documentation for `@trpc/client` can be found [here](https://trpc.io/docs/vanilla)
+
+## Installation
+
+```bash
+# NPM
```
Comment (sachinraja) [inline packages/client/README.md:25]:
```suggestion
# npm
```

File: packages/client/README.md [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+
+```typescript
+import { createTRPCClient, createTRPCClientProxy } from '@trpc/client';
+
+// Importing the router type from the server file
+import type { AppRouter } from './server';
+
+// Initializing the tRPC client
+const client = createTRPCClient<AppRouter>({
+  url: `http://localhost:2022`,
```
Comment (sachinraja) [inline packages/client/README.md:45]:
```suggestion
  url: `http://localhost:3000`,
```
Next's default port

File: packages/client/README.md [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+
+```bash
+# NPM
+npm install @trpc/client
+
+# Yarn
+yarn add @trpc/client
+
+# Pnpm
+pnpm install @trpc/client
```
Comment (sachinraja) [inline packages/client/README.md:32]:
```suggestion
pnpm add @trpc/client
```

File: packages/next/README.md [hunk 5]
```diff
@@ -1,3 +1,86 @@
...
+
+<p align="center">
+  <!-- TODO: replace with new version GIF -->
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
+</p>
+
 # `@trpc/next`
 
-https://trpc.io
+> The `@trpc/next` package is responsible for connecting a trpc server to a Next.js application.
```
Comment (sachinraja) [inline packages/next/README.md:16]:
```suggestion
> Connect a tRPC router to Next.js.
```

File: packages/next/README.md [hunk 6]
```diff
@@ -1,3 +1,86 @@
...
+npm install @trpc/next @trpc/react react-query
+
+# Yarn
+yarn add @trpc/next @trpc/react react-query
+
+# Pnpm
+pnpm install @trpc/next @trpc/react react-query
+```
+
+## Basic Example
```
Comment (sachinraja) [inline packages/next/README.md:35]: I'm unsure about having these examples here to be honest. It's another thing that has to be kept up to date alongside the docs.

File: packages/react/README.md [hunk 7]
```diff
@@ -1,3 +1,90 @@
...
+<p align="center">
+  <!-- TODO: replace with new version GIF -->
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
+</p>
+
 # `@trpc/react`
 
-https://trpc.io
\ No newline at end of file
+> The `@trpc/react` package is responsible for connecting a trpc server to a react application.
```
Comment (sachinraja) [inline packages/react/README.md:16]:
```suggestion
> Connect a tRPC server to React.
```

File: packages/server/README.md [hunk 8]
```diff
@@ -1,3 +1,83 @@
...
+<p align="center">
+  <!-- TODO: replace with new version GIF -->
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
+</p>
+
 # `@trpc/server`
 
-https://trpc.io
\ No newline at end of file
+> The `@trpc/server` package is responsible for creating trpc APIs on the server.
```
Comment (sachinraja) [inline packages/server/README.md:16]:
```suggestion
> Create tRPC routers and connect them to a server.
```

File: packages/client/README.md [hunk 9]
```diff
@@ -1,3 +1,61 @@
...
+
+<p align="center">
+  <!-- TODO: replace with new version GIF -->
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
+</p>
+
 # `@trpc/client`
 
-https://trpc.io
+> The `@trpc/client` package is responsible for communicating with a tRPC server on the client side.
```
Comment (sachinraja) [inline packages/client/README.md:16]:
```suggestion
> Communicate with a tRPC server on the client side.
```

**Review by sachinraja — CHANGES_REQUESTED** [review]

File: packages/next/README.md (hunk 6 above)
Comment (ixahmedxi) [reply packages/next/README.md:35]: I think it's very minimal and small that it won't be hard to keep updated and gives the user some kind of gist about the package.

**Review by sachinraja — APPROVED** [review]

**Review by KATT — APPROVED** [review]
Thanks for doing this!

**Comment by KATT** [comment]
@all-contributors add @ixahmedxi for docs

**Pull request merged**